tools/performance-benchmarking/src/md/0010-content.md

```diff
@@ -0,0 +1,916 @@
+`benchcomp` allows you to:
+
+* Run two or more sets of benchmark suites under different configurations;
+* declaratively define visualizations to be generated from the resulting data;
+* run the tool with sensible defaults, or hook into every stage of the pipeline when you need more flexibility.
+
+This documentation contains three sections.
+The [user walkthrough](#user-walkthrough) takes you through the process of setting up an entirely new benchmark run and dashboard.
+The [developer reference](#developer-reference) describes `benchcomp`'s architecture and the different data formats that it uses, enabling you to author a benchmark run and dashboard of your own.
+The [visualization viewer guide](#benchcomp-visualization-viewer-guide) describes how to use dashboards that `benchcomp` generates.
+Although dashboards are highly customizable, the guide describes the common elements.
+
+
+# User Walkthrough
+
+This section presents a single user story, showing how the user can quickly get started with a simple benchmark comparison run, through to a more complex setup with extensive customization.
+This section explains the syntax in prose without attempting to exhaustively document it; the [Development Reference](#developer-reference) below is a full reference for the syntax.
+
+
+## Comparing two variants
+
+Alice wants to compare the performance of two versions of CBMC: the latest release, with and without a new optimization that she's implemented.
+Initially, Alice tries to do this entirely using `benchcomp`'s built-in parsers and filters.
+She performs the following steps:
+
+* Create a directory containing all AWS CBMC proofs, with a top-level script that runs all of them called `run-all-cbmc-proofs.py`
```

Comment:
Should we expand a bit on what this script is? Doesn't benchcomp rely on the output of this script?